### ISSUE TITLE
why no softmax

### ISSUE THREAD

**Issue by YourGc** [miraclewkf/SENet-PyTorch#6]
hello,
i wanna try my custom datasets which has 21classes (20+1) ,but i finlly find that output Tensor is a logical value , why no SOFTMAX ?
